package.json

```diff
@@ -30,7 +30,8 @@
   },
   "license": "MIT",
   "dependencies": {
-    "@types/bluebird": "3.5.36",
+    "@babel/polyfill": "7.12.1",
+    "@babel/runtime": "7.15.4",
```

Comment:
Why do you have both `@babel/polyfill` and `babel-polyfill` (in devDeps)?

Reply:
Removed both. [Turns out they are not required](https://babeljs.io/docs/en/babel-polyfill)